### DISCUSSION TITLE
Background Services of Notification

### DISCUSSION THREAD

**Discussion by weilerloo** [thudugala/Plugin.LocalNotification#385] [Q&A] [answered]
I was wondering if the notification will be sent at the scheduled time after the app is closed. I had try a few time of my program and it is showing on the screen during the app open, or in tabbed(background). After it is closed, it will never shows. anyone have any idea?

**Comment by thudugala** [comment, marked as answer]
@weilerloo Yes, this plugin does support scheduled notifications after the app is closed. but 
 https://github.com/thudugala/Plugin.LocalNotification/wiki/Scheduled-Android-notifications

**Reply by luis95gr** [reply]
Is there a special permission or something that the user can enable to receive scheduled notifications after the app is closed?

**Reply by thudugala** [reply]
@luis95gr please follow the permission settings in this doc
 https://github.com/thudugala/Plugin.LocalNotification/wiki/Usage-10.0.0--.Net-MAUI

**Comment by hassanraf** [comment]
Hi @thudugala ,

I have been using your plugin and so far it has worked fine for all my needs. However I have lately discovered that after I reboot my Android phone , I cannot receive the notification ( seems like the scheduler does not start). Once I open the app and then close it I can receive them as normal. Can you please guide? I have included all permissions and have the latest nuget package installed. Thanks

**Reply by thudugala** [reply]
Have you setup 

```xml
<uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
```

**Reply by hassanraf** [reply]
I have it in the manifest and am now trying to implement a Bootreceiver inheriting BroadcastReceiver. ( Do I have to do this step of implementing the receiver because your documentation doesn't suggest it) . also I am using Reminder as notification type, should I change it to service? @thudugala

**Reply by thudugala** [reply]
@hassanraf this already implemented in the plugin https://github.com/thudugala/Plugin.LocalNotification/blob/master/Source/Plugin.LocalNotification/Platforms/Android/ScheduledAlarmReceiver.cs

**Reply by hassanraf** [reply]
@thudugala yes I had a look at your source code and saw that it was the case. After investigating further it turned out that I was using Xiaomi for testing which disables auto start. I have put a workaround for now which is to prompt the user to enable the settings by taking them to the page upon start up. Thanks for your replies 👍
